### ISSUE TITLE
Failing to download sysdig-probe-0.24.0-x86_64-3.10.0-862.6.3.el7.x86_64-aa76ad50d3f3599996ad932dcaa99003.ko

### ISSUE THREAD

**Issue by beauhoyt** [draios/sysdig#1243]
Looks like it doesn't exist or permission issue when trying to download:

`sysdig-probe-0.24.0-x86_64-3.10.0-862.6.3.el7.x86_64-aa76ad50d3f3599996ad932dcaa99003.ko`

Problem:
``` 
$ sudo sysdig-probe-loader
* Unloading sysdig-probe, if present
* Running dkms install for sysdig
Error! echo
Your kernel headers for kernel 3.10.0-862.6.3.el7.x86_64 cannot be found at
/lib/modules/3.10.0-862.6.3.el7.x86_64/build or /lib/modules/3.10.0-862.6.3.el7.x86_64/source.
* Running dkms build failed, couldn't find /var/lib/dkms/sysdig/0.24.0/build/make.log
* Trying to load a system sysdig-probe, if present
* Trying to find precompiled sysdig-probe for 3.10.0-862.6.3.el7.x86_64
Found kernel config at /boot/config-3.10.0-862.6.3.el7.x86_64
* Trying to download precompiled module from https://s3.amazonaws.com/download.draios.com/stable/sysdig-probe-binaries/sysdig-probe-0.24.0-x86_64-3.10.0-862.6.3.el7.x86_64-aa76ad50d3f3599996ad932dcaa99003.ko
Download failed, consider compiling your own sysdig-probe and loading it or getting in touch with the sysdig community
```

Though it works for a slightly older version:
```
$ sudo sysdig-probe-loader
* Unloading sysdig-probe, if present
rmmod: ERROR: Module sysdig_probe is not currently loaded
* Running dkms install for sysdig
Error! echo
Your kernel headers for kernel 3.10.0-862.6.3.el7.x86_64 cannot be found at
/lib/modules/3.10.0-862.6.3.el7.x86_64/build or /lib/modules/3.10.0-862.6.3.el7.x86_64/source.
* Running dkms build failed, couldn't find /var/lib/dkms/sysdig/0.23.1/build/make.log
* Trying to load a system sysdig-probe, if present
* Trying to find precompiled sysdig-probe for 3.10.0-862.6.3.el7.x86_64
Found kernel config at /boot/config-3.10.0-862.6.3.el7.x86_64
* Trying to download precompiled module from https://s3.amazonaws.com/download.draios.com/stable/sysdig-probe-binaries/sysdig-probe-0.23.1-x86_64-3.10.0-862.6.3.el7.x86_64-aa76ad50d3f3599996ad932dcaa99003.ko
Download succeeded, loading module
```
works: `sysdig-probe-0.23.1-x86_64-3.10.0-862.6.3.el7.x86_64-aa76ad50d3f3599996ad932dcaa99003.ko`
fails: `sysdig-probe-0.24.0-x86_64-3.10.0-862.6.3.el7.x86_64-aa76ad50d3f3599996ad932dcaa99003.ko`

**Comment by beauhoyt** [comment]
Seems to be fixed now. Thank you 👍

**Comment by mfdii** [comment]
We had just released 0.24.0 when it was failing for you. The kernel modules
hadn't made it to the s3 bucket yet.

**Comment by beauhoyt** [comment]
@mfdii Ah I see. Thank you for the explanation! 😄
